tests/unit/components/TermBox.spec.ts

```diff
@@ -0,0 +1,136 @@
+import Vuex from 'vuex';
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+import TermBox from '@/components/TermBox.vue';
+import store from '@/store/index';
+import {
+	NS_ENTITY,
+	NS_USER,
+	NS_LANGUAGE,
+} from '@/store/namespaces';
+import { ENTITY_INIT } from '@/store/entity/mutationTypes';
+import { LANGUAGE_INIT } from '@/store/user/mutationTypes';
+import { LANGUAGE_TRANSLATION_UPDATE } from '@/store/language/mutationTypes';
+import FingerprintableEntity from '@/datamodel/FingerprintableEntity';
+import LanguageTranslations from '@/datamodel/LanguageTranslations';
+
+const localVue = createLocalVue();
+const langDe = 'Deutsch';
+const labelDe  = 'All I know';
+const descriptionDe = 'Jakob mag potatoes.';
+const aliasesDe = [ 'Antwort auf alles', 'Ihr kennt ja nicht einmal die Frage!' ];
+localVue.use( Vuex );
+
+store.commit(
+`${NS_ENTITY}/${ENTITY_INIT}`,
+	new FingerprintableEntity(
+		'Q42',
+		{
+			de: {
+				language: 'de',
+				value: labelDe,
+			},
+		},
+		{
+			de: {
+				language: 'de',
+				value: descriptionDe,
+			},
+		},
+		{
+			de: [
+				{
+					language: 'de',
+					value: aliasesDe[0],
+				},
+				{
+					language: 'de',
+					value: aliasesDe[1],
+				},
+			],
+		},
+	),
+);
+
+store.commit(
+	`${NS_LANGUAGE }/${LANGUAGE_TRANSLATION_UPDATE}`,
+	{
+		de: {
+			de: langDe,
+			en: 'Englisch',
+		},
+		en: {
+			de: 'German',
+			en: 'English',
+		},
+	} as LanguageTranslations,
+);
+
+describe( 'TermBox.vue', () => {
+	it( 'renders the language label', () => {
+		store.commit( `${NS_USER}/${LANGUAGE_INIT}`, 'de' );
+
+		const wrapper = shallowMount( TermBox, { store, localVue } );
+		expect( wrapper.text() ).toContain( langDe );
```

Comment:
```suggestion
		expect( wrapper.find( '.wikibase-termbox-language' ).text() ).toBe( langDe );
```

would be my preferred approach.
`.text()` of the complex wrapper is really just a bunch of innerHtml and asserting against this is a game of chance.